README.md

```diff
@@ -19,8 +19,8 @@ export { default } from 'ember-strict-resolver';
 
 # Development of the addon
 
-* `git clone <repository-url>` this repository
-* `cd strict-resolver`
+* `git clone ember-strict-resolver`
```

Comment:
This needs to be tweaked a bit...

Reply:
There's an issue for re-writing the README, but are you saying there's something wrong specifically with the change I made? As I don't see it...

Reply:
That isn’t a valid git clone URL. Should likely be `git@github.com:stefanpenner/ember-strict-resolver.git`

Reply:
Doh!